CBLClient/Apps/CBLTestServer-Dotnet/TestServer.NetCore/Program.cs

```diff
@@ -50,8 +51,18 @@ static void Main(string[] args)
             Couchbase.Lite.Support.NetDesktop.Activate();
             Extend();
 
-            Couchbase.Lite.Support.NetDesktop.EnableTextLogging("TextLogging");
-            Database.SetLogLevel(Logging.LogDomain.All, Logging.LogLevel.Info);
+            Database.Log.Console.Level = Logging.LogLevel.Debug;
+            Database.Log.Console.Domains = Logging.LogDomain.All;
+            String logPath = Path.Combine(Directory.GetCurrentDirectory(), "LogTestLogs");
```

Comment:
The logic is consistent and is enforced based on platform. The current directory format is different across all platform so we can't have same format. Current directory is the path for installed app and hence it will always be there.